openshift/mongo.helm.yml

```diff
@@ -0,0 +1,25 @@
+fullnameOverride: hcap-mongo
+persistence:
+  size: 4Gi
+
+podSecurityContext:
+  enabled: false
+  fsGroup: 1008111001
```

Comment:
Are these ids or do they have a specific meaning?

Reply:
The ids can be removed, I believe. We're not enabling them anyways.